MaxText/train.py

```diff
@@ -211,13 +212,30 @@ def train_loop(config, state=None):
   Returns:
 
   """
+
+  monitoring_enabled = config.enable_cloud_monitoring
+
+  if monitoring_enabled:
+    monitoring_api.create_custom_metric('checkpointing_init_start', "Checkpointing Initialization Start")
+    monitoring_api.create_custom_metric('checkpointing_init_end', "Checkpointing Initialization End")
+    monitoring_api.create_custom_metric('checkpoint_test_run_start', "Checkpointing Test Run Start")
+    monitoring_api.create_custom_metric('checkpoint_test_run_end', "Checkpointing Test Run End")
+
+  monitoring_api.write_time_series_step('checkpoint_test_run_start', 0, monitoring_enabled)
```

Comment:
I'm a little confused by what is happening here. It looks to me like `write_time_series_step` has arguments in a different order than you're passing them in?

Reply:
I think it may have been an error on a non-latest commit - Fixed.